e2e/utils/waits-utils.ts

```diff
@@ -268,9 +268,11 @@ export async function waitForText(page: Page,
  * It usually indicates the page is ready for user interaction.
  * </pre>
  */
-export async function waitWhileLoading(page: Page, timeout: number = (2 * 60 * 1000)): Promise<void> {
+export async function waitWhileLoading(page: Page, timeout: number = (2 * 60 * 1000), opts: {waitForRuntime?: boolean} = {}): Promise<void> {
```

Comment:
Function can check and wait for page spinner and runtime spinner. But the default is not waiting for runtime spinner.